examples/ethereum/list-stakes/main.go

```diff
@@ -53,12 +63,11 @@ func main() {
 			log.Fatalf("error listing stakes: %s", err.Error())
 		}
 
-		d, err := protojson.Marshal(stake)
+		marshaled, err := json.MarshalIndent(stake, "", "   ")
```

Comment:
using json.MarshalIndent instead of protojson.Marshal might change the output in which case we should update the Readme.

Any reason to do this ? I think I used protojson.Marshal as it did a better job at marshaling enums (I think)

Reply:
Yeh you're totally right `json.MarshalIndent` doesn't correctly handle enums 😞. I moved away from `protojson.Marshal` because it didn't pretty print the results. I just updated it to a version where I think we can get the best of both words (with some extra lines), LMK what you think (or if you know of a more concise way to do this).

Reply:
Maybe try the marshal options ? https://github.com/protocolbuffers/protobuf-go/blob/v1.33.0/encoding/protojson/encode.go#L45-L54

Reply:
Ah, nice! Done 😄.